src/App.js

```diff
@@ -5,11 +5,13 @@ import Login from './pages/Login';
 import Game from './pages/Game';
 import Feedback from './pages/Feedback';
 import Ranking from './pages/Ranking';
+import Settings from './pages/Settings';
 
 export default function App() {
```

Comment:
O que acha de alterar a forma com que esse componente é declarado para padronizar todo o projeto? :smiley:
```suggestion
import React from 'react'

import Game from './pages/Game';
import Feedback from './pages/Feedback';
import Ranking from './pages/Ranking';
import Settings from './pages/Settings';

export default class App extends React.Component {
```